### DISCUSSION TITLE
Same uuid in the db_v2.sqlite3 database

### DISCUSSION THREAD

**Discussion by mikekgr** [rustdesk/rustdesk#908] [General]
Dear Sirs,
examining the db_v2.sqlite3 database of my server, I saw that there is some duplicates of uuid in some of my connected PC. 
I mean there are 3 or 4 PC's that they share the exactly same uuid.
Is there any problem regarding this or it's normal?

Thanks and Best Regards

**Comment by mikekgr** [comment]
Good morning.
Any reply for this topic?
Thanks.

**Comment by rustdesk** [comment]
I wanna know it also.

**Reply by mikekgr** [reply]
Sorry I missed something probably !  You are the developer and you don't know??? If you need something from me, to make your analysis, don't hesitate to ask me.

**Reply by rustdesk** [reply]
You overestimated the developer.
